plugins/callback/foreman.py

```diff
@@ -19,6 +20,26 @@
       - whitelisting in configuration
       - requests (python library)
     options:
+      report_type:
+        description:
+          - endpoint type for reports: foreman or proxy
```

Comment:
```suggestion
          - "endpoint type for reports: foreman or proxy"
```

YAML is picky if you have colons in your strings